src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -712,6 +716,21 @@ private JArray GetConsensusState()
             return protocols;
         }
 
+        [JsonRpcMethod("la_getValidatorsAfterBlock")]
+        public JArray GetValidatorsAfterBlock(string blockTag)
+        {
+            if (blockTag == "pending")
```

Comment:
Shouldn't we handle cases like blockTag doens't belong to {"pending", "latest"}?

Reply:
Sorry did not understand the question

Reply:
We are checking "pending" value only because all other values are processed in  `GetBlockNumberByTag` call @kunalgoyal9